index.html

```diff
@@ -164,53 +176,62 @@ <h3 class="project-title">Multi-Post Stories</h3>
         </section>
     </div>
     <!-- About myself -->
-    <div class="about-myself">
-        <h2 class="about">About Myself</h2>
-        <p class="text">
-            Hello I’m a software developer! I can help you build a product , feature
-            or website Look through some of my work and experience! If you like what
-            you see and have a project you need coded, don’t hestiate to contact me.
-        </p>
-        <h2 class="hex connect">LET’S CONNECT</h2>
-        <ul class="social-media">
-            <li clalss="socials">
-                <a href="#"><img src="images/socials/twitter.svg" /></a>
-            </li>
-            <li clalss="socials">
-                <a href="#"><img src="images/socials/Linkedin.svg" /></a>
-            </li>
-            <li clalss="socials">
-                <a href="#"><img src="images/socials/m.svg" /></a>
-            </li>
-            <li clalss="socials">
-                <a href="#"><img src="images/socials/Vector1.svg" /></a>
-            </li>
-            <li clalss="socials">
-                <a href="#"><img src="images/socials/Vector2.svg" /></a>
-            </li>
-        </ul>
-        <div class="see-project">
-            <a href="#"><button type="button" class="hex">Get my resume</button></a>
-        </div>
+    <div class="about-myself" class="myself">
+        <div class="about-text" class="text-about">
+            <h2 class="about">About Myself</h2>
+            <p class="texts" id="texte" >
+                Hello I’m a software developer! I can help you build a product , feature
+                or website Look through some of my work and experience! If you like what
+                you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
```

Comment:
- [ ] Dear @Christelle-12, for your `about` and `contact me` sections, Please notice there is no blurred spacing on the right and left side of the page on the figma design. your design seems to have one. Please remove this space to have an exceptional design. 

![image](https://user-images.githubusercontent.com/69077061/222879068-9b591e09-0a24-49f7-a5b7-be83910cccde.png)